### DISCUSSION TITLE
config of models folder

### DISCUSSION THREAD

**Discussion by lennart81** [ENHANCE-PET/MOOSE#139] [General]
Hi there,

When I want to run moose on a linux server I get following error:
[Errno 13] Permission denied: '/usr/models'

Can I tell moose to save the model somewhere else?

**Comment by LalithShiyam** [comment]
You can - but it involves quite a bit of hacking. I assume you are using it in a different server where you don't have admin access?

**Reply by lennart81** [reply]
correct. 

I was searching an easy solution - maybe I can ask an admin to download the models?

**Reply by LalithShiyam** [reply]
Yep, just create a venv with admin access and everything should work.
